### PR TITLE
Update the SPDX license enums and lists to version 3.8

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: spdx-utils/src/main/kotlin/SpdxLicense.kt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Copyright (C) 2017-2020 HERE Europe B.V.
```
Comment (fviernau) [inline spdx-utils/src/main/kotlin/SpdxLicense.kt:2]: Wanna add Bosch?

Comment (sschuberth) [reply spdx-utils/src/main/kotlin/SpdxLicense.kt:2]: Well, that code is auto-generated, and the generator was written when I was still at HERE, so I think it's fine.

**Comment by sschuberth** [comment]
Blocked by https://github.com/fsfe/reuse-tool/issues/229.

**Comment by sschuberth** [comment]
Merging despite the unrelated AppVeyor failure as Travis (which runs `reuse lint`) passed, and AppVeyor had passed before the rebasing.

**Pull request merged**